tutorials/hpo-workflows/data.py

```diff
@@ -0,0 +1,105 @@
+import argparse
+import sys
+from pathlib import Path
+from typing import Tuple
+
+from torch.utils.data import Dataset, random_split
+from torchvision import datasets, transforms
+
+from itwinai.components import DataGetter, DataSplitter
+
+data_dir = Path("data")
+
+
+def download_fashion_mnist() -> None:
+    """
+    Download the FashionMNIST dataset using torchvision.
+    """
+    print("Downloading FashionMNIST dataset...")
+    _ = datasets.FashionMNIST(
```

Comment:
In this case, you can just omit the `_ =` and just do `datasets.FashionMNIST(...)` directly.

Reply:
Oh interesting, okay